zendesk_to_weaviate.ipynb

```diff
@@ -0,0 +1,557 @@
+      "metadata": {
+        "id": "sUs3ttXabIyC"
+      },
+      "source": [
+        "## Load Data"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "wUeAEOIjZmb3"
+      },
+      "outputs": [],
+      "source": [
+        "import dlt\n",
+        "from dlt.destinations.weaviate import weaviate_adapter\n",
+        "\n",
+        "from zendesk import zendesk_support\n",
+        "\n",
+        "pipeline = dlt.pipeline(\n",
+        "    pipeline_name=\"zendesk_pipeline\",\n",
+        "    destination=\"weaviate\",\n",
+        ")\n",
+        "\n",
+        "zendesk_source = zendesk_support(load_all=False)\n",
+        "tickets = zendesk_source.tickets\n",
+        "\n",
+        "info = pipeline.run(\n",
+        "    weaviate_adapter(\n",
+        "        tickets,\n",
+        "        vectorize=[\"subject\", \"description\"],\n",
+        "    ),\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "rI_dPWk4ZzG9",
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "outputId": "cb736690-6d7a-482f-c7a5-8373d3f47093"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Pipeline zendesk_pipeline completed in 10.23 seconds\n",
+            "1 load package(s) were loaded to destination weaviate and into dataset None\n",
+            "The weaviate destination used https://demo-colab-ng73n7k8.weaviate.network location to store data\n",
+            "Load package 1693572743.108808 is LOADED and contains no failed jobs\n"
+          ]
+        }
+      ],
+      "source": [
+        "print(info)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "And we're done. Let's check out the data."
+      ],
+      "metadata": {
+        "id": "rYqwjZwMaiQL"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Query Data\n",
+        "\n",
+        "You need to replace the `url` and `api_key` below with your Weaviate instance credentials."
+      ],
+      "metadata": {
+        "id": "0Pdf4S2qoiQB"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "import pandas as pd\n",
+        "import weaviate\n",
+        "client = weaviate.Client(\n",
+        "    url='https://YOUR_WEAVIATE_URL',\n",
+        "    auth_client_secret=weaviate.AuthApiKey(\n",
+        "        api_key='YOUR_WEAVIATE_API_KEY'\n",
+        "    ),\n",
+        ")\n",
+        "response = (\n",
+        "      client.query\n",
+        "      .get(\"Tickets\", [\"subject\", \"description\", \"created_at\"])\n",
+        "      .do()\n",
+        ")\n",
+        "\n",
+        "response_to_table(response, 'Tickets', [\"subject\", \"description\", \"created_at\"])"
```

Comment:
I can't find response_to_table function, it's confusing

Reply:
Good point, I think I've lost it along the way.

Reply:
Hmm, I have it in the colab version: https://colab.research.google.com/drive/1nxaIOUtuW7mdHcHZlHQyn578bvSN2pq. Might be an export issue.